conformance/utils/kubernetes/apply.go

```diff
@@ -55,41 +56,112 @@ type Applier struct {
 
 	// ControllerName will be used as the spec.controllerName when applying GatewayClass resources
 	ControllerName string
+
+	namespaceLabels map[string]string
```

Comment:
We already have a nearly identical `NamespaceLabels` attribute on this struct, do we actually need to make a distinction here? If so, can we add comments explaining why?

Reply:
I meant to make all fields private, I think `NamespaceLabels` crept back up in a rebase. I introduced the `New` pattern to avoid the writing to a `nil` map issue, then more public fields were introduced. I'll just get rid of the `New` pattern and deal with the `nil` map instead.